ophyd/controls/ophydobj.py

```diff
@@ -319,6 +319,17 @@ def check_value(self, value, **kwargs):
         '''
         pass
 
+    @property
+    def state(self):
+        return {}
+
+    def configure(self, state=None):
+        # does nothing; subclasses can override if configuration is possible
+        return self.state, self.state
+
+    def deconfigure(self):
+        return self.state
+
```

Comment:
In any case, this needs to properly deal with the MI from `Detector` (which coming back to it, all of the methods it provides as a mix in now live in `ophydobj` anyway so it can probably go away).

If it goes on `SignalGroup` then we will need to re-implement it for the `Signal`/`EpicsSignal` chain.